test/unit/controllers/declaration/TotalNumberOfItemsControllerSpec.scala

```diff
@@ -33,89 +32,150 @@ import views.html.declaration.total_number_of_items
 
 class TotalNumberOfItemsControllerSpec extends ControllerSpec with OptionValues {
 
-  val mockTotalNumberOfItemsPage = mock[total_number_of_items]
+  def theResponseForm(mockTotalNumberOfItemsPage: total_number_of_items): Form[TotalNumberOfItems] = {
+    val captor = ArgumentCaptor.forClass(classOf[Form[TotalNumberOfItems]])
+    verify(mockTotalNumberOfItemsPage).apply(any(), captor.capture())(any(), any())
+    captor.getValue
+  }
 
-  val controller = new TotalNumberOfItemsController(
-    mockAuthAction,
-    mockJourneyAction,
-    navigator,
-    stubMessagesControllerComponents(),
-    mockTotalNumberOfItemsPage,
-    mockExportsCacheService
-  )(ec)
+  trait SetUp {
```

Comment:
Because about 3-5 months ago we decided to go with traits. Follows convention.
I am happy to discuss this with the team again, although it will be a bit annoying as we keep reviewing these tests setups.